class.jetpack-jitm.php

```diff
@@ -180,9 +236,9 @@ function manage_pi_msg() {
 	 */
 	function editor_msg() {
 		global $typenow;
-		if ( current_user_can( 'jetpack_manage_modules' ) && current_user_can( 'manage_options' ) ) {
+		if ( current_user_can( 'manage_options' ) ) {
 			$normalized_site_url = Jetpack::build_raw_urls( get_home_url() );
-			$editor_dismissed = isset( self::$jetpack_hide_jitm['editor'] );
+			$editor_dismissed = isset( self::$jetpack_hide_jitm[ 'editor' ] );
```

Comment:
Derek asked me to add the spaces in https://github.com/Automattic/jetpack/pull/3157#discussion-diff-47830073 and before adding them I searched for it and saw that there were using them in other files. But using them without spaces is also used a lot. I just checked WP core and it's the same. Both are used. WordPress PHP guidelines doesn't mention them. So I'm not sure what's the correct one...